core/makeswift/components/product-detail/register.tsx

```diff
@@ -0,0 +1,25 @@
+import { Select, Slot } from '@makeswift/runtime/controls';
+
+import { runtime } from '~/lib/makeswift/runtime';
+
+import { DescriptionType, MakeswiftProductDetail } from './client';
+
+export const COMPONENT_TYPE = 'catalyst-makeswift-product-detail-description';
+
+runtime.registerComponent(MakeswiftProductDetail, {
+  type: COMPONENT_TYPE,
+  label: 'MakeswiftProductDetail (private)',
+  hidden: true,
+  props: {
+    descriptionType: Select({
+      label: 'Description',
+      options: [
+        { label: 'From BC (plain text)', value: DescriptionType.PlainText },
+        { label: 'From BC (rich text)', value: DescriptionType.RichText },
```

Comment:
I know this is what @apledger suggested in the thread, but "From BC..." feels off to me. May be "From the catalog ..."?

Reply:
Changed to "Catalog (plain text)" etc.